smartseq2_single_sample/ss2_checker.wdl

```diff
@@ -0,0 +1,15 @@
+task SmartSeq2Checker {
+  File rna_metrics
+
+  String expected_rna_metrics_hash
+
+  command <<<
+    rna_metrics_hash=$(tail -n +6 ${rna_metrics} | md5sum | awk '{print $1}')
```

Comment:
you might add `sed -e ` before multiple bash script lines.

Reply:
Thanks, I'll add it.